falcon/response.py

```diff
@@ -42,6 +42,10 @@
     'Please use Response.set_stream() or Response.content_length instead.'
 )
 
+_ADD_LINK_REMOVED_MSG = (
```

Comment:
Could you also rename this variable to reflect its meaning more accurately? Maybe `_ADD_LINK_DEPRECATED_MSG`?